test/auth_cluster/test_auth_no_quorum.py

```diff
@@ -54,3 +55,44 @@ async def test_auth_no_quorum(manager: ManagerClient) -> None:
             auth_provider=PlainTextAuthProvider(username=role, password=role))
     names = [row.role for row in cql.execute(f"LIST ROLES", execution_profile='whitelist')]
     assert(set(names) == set(['cassandra'] + roles))
+
+
+async def trigger_snapshot(manager: ManagerClient, server: ServerInfo) -> None:
+    cql = manager.get_cql()
+    host = cql.cluster.metadata.get_host(server.ip_addr)
+    group0_id = (await cql.run_async(
+        "select value from system.scylla_local where key = 'raft_group0_id'",
+        host=host))[0].value
+    await manager.api.client.post(f"/raft/trigger_snapshot/{group0_id}", host=server.ip_addr)
+
+"""
+Tests raft snapshot transfer of auth data.
+"""
+@pytest.mark.asyncio
+async def test_auth_raft_snapshot_transfer(manager: ManagerClient) -> None:
+    servers = await manager.servers_add(3)
+
+    cql = manager.get_cql()
+    await wait_for_cql_and_get_hosts(cql, servers, time.time() + 60)
+    await manager.servers_see_each_other(servers)
+
+    roles = ["ro" + unique_name() for _ in range(10)]
+    for role in roles:
+        # if not exists due to https://github.com/scylladb/python-driver/issues/296
+        cql.execute(f"CREATE ROLE IF NOT EXISTS {role}")
+
+    await trigger_snapshot(manager, servers[0])
```

Comment:
When adding a node to the cluster, the current leader will send its latest snapshot to it. It is not guaranteed that server 0 will be the leader, you can create the snapshot on all nodes just to be sure.

Reply:
switch `servers_add(3)` to `servers_add(1)` should solve this